src/status_im/chat/models/input.cljs

```diff
@@ -6,6 +6,7 @@
             [status-im.chat.constants :as chat.constants]
             [status-im.ethereum.json-rpc :as json-rpc]
             [status-im.chat.models :as chat]
+            [status-im.utils.debounce :as debounce]
```

Comment:
isn't used?

Reply:
yeah, was trying something. This is not needed.